tests/utils/helpers.js

```diff
@@ -4,13 +4,17 @@ const axios = require('axios');
 const verifyUrl = async (page, expectedUrl) =>
   expect(page.url()).toEqual(expectedUrl);
 
-const verifyRedirectUrl = async (page, baseUrl, expectedUrl) => {
-  await page.goto(baseUrl, { waitUntil: 'networkidle' });
-  expect(page.url()).toEqual(expectedUrl);
+const verifyRedirectUrl = async (base, expected, options) => {
+  options = Object.assign({ status: 200 }, options);
```

Comment:
nit, let's not get in the habit of modifying arguments. Also, we can use the spread operator for better readability, IMO.
```suggestion
  const parsedOptions = { ...options, status: 200 };
```